packages/ember-glimmer/lib/environment.js

```diff
@@ -62,6 +62,29 @@ function wrapClassAttribute(args) {
   return args;
 }
 
+function wrapClassBindingAttribute(args) {
+  let hasClassBinding = args.named.has('classBinding');
+
+  if (hasClassBinding) {
+    let { value , type } = args.named.at('classBinding');
+    if (type === 'value') {
+      let [ prop, truthy, falsy ] = value.split(':');
+      if (prop === '') {
+        let syntax = HelperSyntax.fromSpec(['helper', ['-class'], [truthy], null]);
+        args.named.add('class', syntax);
```

Comment:
I think the issue with this is that you are causing some extra work being done. Consider that you are taking the component call site and going from:

```handlebars
{{foo-bar classBinding=foo}}
```
to
```handlebars
{{foo-bar classBinding=foo class=(-class 'truthy') }}
```

What I think you should do is probably replacing at the callsite:

```handlebars
{{foo-bar classBinding=(-class 'truthy')}}
```

In the end it may not matter but from a debugging point of view it may be weird.